### PR TITLE
✨Monitoring tools: show more information about computational clusters

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
- [x] added Makefile to install dependencies
- [x] adds separate requirement file
- [x] use dask to connect with computational clusters and get informations about known jobs, and currently processing ones
- [x] adds graylog permalinks to facilitate debugging

new entries look like:
image

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]
Thanks, i will give a try today.

File: scripts/maintenance/computational-clusters/Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.DEFAULT_GOAL := install
+
+SHELL := /bin/bash
```
Comment (bisgaard-itis) [inline scripts/maintenance/computational-clusters/Makefile:3]: Out of curiosity: Why do you set this variable? It looks to me like it is not used below

**Review by bisgaard-itis — APPROVED** [review]
Cool! Thanks a lot

File: scripts/maintenance/computational-clusters/Makefile (hunk 1 above)
Comment (sanderegg) [reply scripts/maintenance/computational-clusters/Makefile:3]: SHELL? this defines what shell the GNU Make shall use. by default it uses posix shell or ```sh```

Comment (bisgaard-itis) [reply scripts/maintenance/computational-clusters/Makefile:3]: aha, good to know. I still have a lot of Make to learn. Thanks!

**Pull request merged**